### DISCUSSION TITLE
Unterstützung für Zeitmarken

### DISCUSSION THREAD

**Discussion by BabyFnord** [kaixxx/noScribe#12] [Ideas]
Hallo @kaixxx,

vielen Dank für die admin-freie Installationsoption, die funktioniert perfekt! So gut, wie die Transkription rauskommt:
Bisher habe ich die Nachbearbeitung fertiger Transkripte in F4/F5 (audiotranskription.de) erledigt, da Zeitmarken es dort sehr leicht machen, die Stellen anzuspringen. Zeitmarken erlauben es auch unabhängig von Word (think Libre Office), manuell, die Audiodatei zu verwenden. Bei mir funktioniert die Audio-Wiedergabe ohnehin nicht in Word.

Wäre es möglich, wahlweise Zeitmarken nach dem Muster `#HH:MM:SS-s#` in das Transkript zu integrieren?

Bestes, BF

**Comment by kaixxx** [comment]
Ich überlege, in einer zukünftigen Version die Option zu integieren, das Transkript auch als normale Textdatei ausgeben zu lassen (vor allem für jene, die kein Word nutzen). Da würde ich dann auch Zeitmarken in den Text integrieren, danke für die Anregung!

**Reply by BabyFnord** [reply]
Nachtrag: Es wäre ausreichend, wenn die Zeitmarke `#HH:MM:SS-s#` jeweils beim Sprecher:innenwechsel gesetzt würde.

**Comment by kaixxx** [comment]
Nachtrag: Prinzipiell sind die Zeitmarken auch jetzt schon einsehbar. Du wirst in deinem Transkript viele Textmarken mit dem prefix "ts_" finden (Einfügen > Textmarke). Die Zeitstempel in der Audiodatei ist im Namen codiert, in Millisekunden. "ts_28320_29200" bedeutet: Der Text in dieser Textmarke entspricht in der Audiodatei Millisekunde 28320 bis 29200.

**Comment by BabyFnord** [comment]
Danke für die Antwort und für das in Betracht ziehen der Zeitmarken. Zeitmarken wie `#HH:MM:SS-s#` sind schon verständlicher als Millisekunden 😊, insbesondere bei langen Transkripten.
